### PR TITLE
separate components, container and link front-end'

### PR DESCRIPTION
**Pull request by yuda1124**

### CODE REVIEW

File: shared/components/ListWrapper.js [hunk 1]
```diff
@@ -35,20 +27,20 @@ const styles = StyleSheet.create({
...
+  //     card: {
+  //       title: 'Title',
+  //       description: 'desc'
+  //     }
+  //   };
+  // }
+  //
+  // handleTextChange(event) {
+  //   this.setState({zip: event.nativeEvent.text});
+  // }
```
Comment (ochanje210) [inline shared/components/ListWrapper.js]: delete unnecessary comments later

File: shared/components/ListWrapper.js [hunk 2]
```diff
@@ -35,20 +27,20 @@ const styles = StyleSheet.create({
 });
```
Comment (ochanje210) [inline shared/components/ListWrapper.js]: unused styles should be deleted?

File: shared/containers/listLayout.js [hunk 3]
```diff
@@ -1,37 +1,25 @@
-'use strict';
```
Comment (ochanje210) [inline shared/containers/listLayout.js]: we actually don't need this strict use

File: shared/containers/listLayout.js [hunk 4]
```diff
@@ -1,37 +1,25 @@
...
+  return {
+    dataSource: state.list.dataSource
+  };
 };
 
-ListLayout.defaultProps = {
-
+const mapDispatchToProps = (dispatch) => {
+  return {
+    //actions: bindActionCreators(listActions, dispatch)
```
Comment (ochanje210) [inline shared/containers/listLayout.js]: unnecessary comment

File: shared/containers/listLayout.js [hunk 5]
```diff
@@ -1,37 +1,25 @@
...
+      getAllItems: () => {
+        return dispatch(getAllItems());
+      }
```
Comment (ochanje210) [inline shared/containers/listLayout.js]: indentation

**Comment by ochanje210** [comment]
overall looks good
LGTM

**Comment by mun625** [comment]
LGTM

**Pull request merged**